operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/InformerConfig.java

```diff
@@ -21,56 +24,60 @@
   String name() default NO_VALUE_SET;
```

Comment:
This does not makes too much sense for the controller either :(

Reply:
I wondered about this, actually, if we could somehow reuse the same config for the controller and some other event source but yes, I agree that this doesn't make much sense for the controller.

The problem is that we cannot inherit between annotations so either:  

- we have separate annotations that do more or less the same thing with code to process each that will again more or less do the same thing, which will cause quite a big maintenance load
- we accept that some of the setup doesn't make sense for both, add proper safeguards and live with it
-  we extract the fields that don't make sense in a different annotation for event sources that will enclose an `InformerConfig` field but that adds yet another level of annotations, which would be less than ideal.

Reply:
Exactly, imo 1. or 2.
Not sure if that is that much more code to have a `@ControllerInformer`. 

Or alternative is also just simply stick with the current approach and don't merge this PR.

Reply:
I think I would prefer 2. at this point, with more clean up on the naming, it should be much better, I just wanted to control the blast radius at this point :)